### PR TITLE
PS-7531 Add links for SELinux and AppArmor docs

### PR DESCRIPTION
**Pull request by patrickbirch**
modified:   source/security/apparmor.rst
	modified:   source/security/selinux.rst

### CODE REVIEW

File: doc/source/security/apparmor.rst [hunk 1]
```diff
@@ -8,161 +8,173 @@ The operating system has a Discretionary Access Controls (DAC) system. AppArmor
...
+If needed, specify the program's path in the command:
 
-.. sourcecode:: bash
+    .. sourcecode:: bash
 
-    $ sudo aa-complain /sbin/<program>
+        $ sudo aa-complain /sbin/<program>
 
-If the profile is not in stored in /etc/apparmor.d, use the following command:
+If the profile is not in stored in the default location, /etc/apparmor.d, use the following command:
```
Comment (nastena1606) [inline doc/source/security/apparmor.rst]: One "in" preposition seems to be excessive

File: doc/source/security/apparmor.rst [hunk 2]
```diff
@@ -8,161 +8,173 @@ The operating system has a Discretionary Access Controls (DAC) system. AppArmor
...
+        $ sudo aa-complain /path/to/profiles/<program>
 
 .. _enforce-one:
 
-Change a Profile to Enforce mode
+Switch a Profile to Enforce mode
 ---------------------------------
 
-To change a profile to enforce mode and the program is in your path:
+To switch a profile to the enforce mode when the program is in your path with this command:
```
Comment (nastena1606) [inline doc/source/security/apparmor.rst]:
```suggestion
Switch a profile to the enforce mode when the program is in your path with this command:
```

File: doc/source/security/apparmor.rst [hunk 3]
```diff
@@ -206,193 +218,197 @@ Install the mysqld profile with the following procedure:
...
-    datadir    = /var/lib/mysql
-    log-error    = /*custom-log-dir*/mysql/error.log
+        [mysqld]
+        pid-file    = /var/run/mysqld/mysqld.pid
+        socket        = /var/run/mysqld/mysqld.sock
+        datadir    = /var/lib/mysql
+        log-error    = /*custom-log-dir*/mysql/error.log
 
-Verify the directory exists.
+Verify the custom directory exists.
```
Comment (nastena1606) [inline doc/source/security/apparmor.rst:319]:
```suggestion
Verify that the custom directory exists.
```

File: doc/source/security/selinux.rst [hunk 4]
```diff
@@ -34,24 +37,20 @@ Most SELinux policy rules are based on the type or domain.
 List SELinux Types or Domains associated with files
 ----------------------------------------------------
 
-For Type Enforcement each policy module,  are used for the interactions between processes and resources. SELinux type names often end with a ``_t``. 
-
-To view the SELinux contexts, add the ``-Z`` switch to many of the utilities. For example, to view the SELinux types associated with MySQL directories and files, run the following command:
+The security property that SELinux relies on is the Type security property. The type name often end with a ``_t``. A group of objects with the same type security value belong to the same domain.
```
Comment (nastena1606) [inline doc/source/security/selinux.rst]:
```suggestion
The security property that SELinux relies on is the Type security property. The type name often ends with a ``_t``. A group of objects with the same type security value belongs to the same domain. 
```
group belongs, right?

File: doc/source/security/apparmor.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/security/apparmor.rst]: Removed - I wanted to be sure the reader could find the default location. It's mentioned in an earlier sentence.

File: doc/source/security/apparmor.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/security/apparmor.rst]: Changed. 
I was being cautious about being too imperative.

File: doc/source/security/apparmor.rst (hunk 3 above)
Comment (patrickbirch) [reply doc/source/security/apparmor.rst:319]: The "that" can safely be omitted.

File: doc/source/security/selinux.rst (hunk 4 above)
Comment (patrickbirch) [reply doc/source/security/selinux.rst]: Fixed.

**Review by nastena1606 — APPROVED** [review]
LGTM

**Pull request merged**